File: lib-model/src/main/java/com/minesec/msa/client/sdk/payment/model/DownloadEmvParamDto.java
package com.minesec.msa.client.sdk.payment.model;

import java.io.Serializable;

import lombok.Getter;
import lombok.Setter;

/**
 * @author eric.song
 * @since 2022/8/26 11:30
 */
@Getter
@Setter
public class DownloadEmvParamDto implements Serializable {

    private String terminalType;

    private String terminalCapability;

    private String terminalAddiCapability;

    private String terminalCountryCode;

    private String txnCurrencyCode;

    private String amexReaderCapability;

    private String amexExReaderCapability;

    private String amexFloorLimit;

    private String emvFlags;

    private String ifdSerialNumber;
}

File: lib-model/src/main/java/com/minesec/msa/client/sdk/management/model_k/DownloadEMVParamDto_.kt
package com.minesec.msa.client.sdk.management.model_k

import java.io.Serializable


/**
 * @author eric.song
 * @since 2024/6/19 15:04
 */
data class DownloadEMVParamDto_(
    var terminalType: String,
    var terminalCapability: String,
    var terminalAddiCapability: String,
    var terminalCountryCode: String,
    var txnCurrencyCode: String,
    var amexReaderCapability: String,
    var amexExReaderCapability: String,
    var amexFloorLimit: String,
    var emvFlags: String,
    var ifdSerialNumber: String
) : Serializable
